### PR TITLE
Upgrade to aws sdk v3

### PR DESCRIPTION
**Pull request by Amndeep7**

### CODE REVIEW

**Review by ChrisHinchey — APPROVED** [review]
After demo and reviewing code it looks good to me

**Review by aaronlippold — CHANGES_REQUESTED** [review]
Other than making sure we have the right environment variable or flag interface for the AWS credentials should they need to override them and allowing for the insecure flag or environment variable for verify SSL Certs it looks pretty straightforward.

File: apps/frontend/src/components/global/upload_tabs/aws/S3Reader.vue [hunk 1]
```diff
@@ -134,35 +133,40 @@ export default class S3Reader extends Vue {
...
   /** If the user tries to login by going to MFA, first check that the account is valid */
-  handle_goto_mfa() {
+  handleGotoMfa() {
     // Attempt to assume role based on if we've determined 2fa necessary
     // Don't need the duration to be very long
-    get_session_token(this.accessToken, this.secretToken, this.region, 10).then(
+    getSessionToken(
+      this.accessToken,
+      this.secretToken,
+      this.region || 'us-east-1',
```
[the remark this answers is not in the document]
Comment (Amndeep7) [reply apps/frontend/src/components/global/upload_tabs/aws/S3Reader.vue:148]: It is documented to the degree that the input box on the modal says that the default is us-east-1

[the remark this answers is not in the document]
Comment (Amndeep7) [reply apps/frontend/src/components/global/upload_tabs/aws/S3Reader.vue:148]: image

[the remark this answers is not in the document]
Comment (Amndeep7) [reply apps/frontend/src/components/global/upload_tabs/aws/S3Reader.vue:148]: The other values must be supplied by the user every time.

File: libs/hdf-converters/src/aws-config-mapper.ts [hunk 2]
```diff
@@ -20,21 +24,26 @@ const NAME = 'AWS Config';
...
 export class AwsConfigMapper {
-  configService: AWS.ConfigService;
+  configService: ConfigService;
   issues: Promise<ConfigRule[]>;
   results: ExecJSON.ControlResult[][];
   constructor(
-    options: AWS.ConfigService.ClientConfiguration,
-    verifySSLCertificates = true
+    options: ConfigServiceClientConfig,
+    verifySSLCertificates = true,
```
[the remark this answers is not in the document]
Comment (Amndeep7) [reply libs/hdf-converters/src/aws-config-mapper.ts:32]: The true is just the default value.  The mapper already supports insecure.  We just don't support custom CAs here.

https://github.com/mitre/heimdall2/blob/master/libs/hdf-converters/src/aws-config-mapper.ts#L28
https://github.com/mitre/saf/blob/main/src/commands/convert/aws_config2hdf.ts#L20

[the remark this answers is not in the document]
Comment (Amndeep7) [reply libs/hdf-converters/src/aws-config-mapper.ts:32]: Actually I lied, forgot that I added in the ability to do the CA already.  Look at the line below.

**Review by aaronlippold — APPROVED** [review]

**Pull request merged**